timon/plugins/base.py

```diff
@@ -47,3 +49,22 @@ async def stop_plugin(self):
     async def stop(self):
         raise NotImplementedError(
             "Plugin %s doesn't have a start method implemented", self.name)
+
```

Comment:
line 52 wasn't changed in this MR

But just saw that the stop method complains, that the start method is not implemented.

perhaps fix it in this MR and add to the description.
'fixed an old doc string'